README.md

```diff
@@ -12,7 +12,7 @@ Person B should modify content here.
 
 ## Person C:
 
-Person C should modify content here.
+HELLO!!! added my-secrets.txt
```

Comment:
What's your secret????

Reply:
the secret is: wear sunblock and drink water every day!